gradio/components.py

```diff
@@ -1678,7 +1679,13 @@ def stream(
         )
 
     def as_example(self, input_data: str | None) -> str:
-        return "" if input_data is None else str(utils.abspath(input_data))
+        if input_data is None:
+            return ""
+        elif (
+            self.root_url
+        ):  # If an externally hosted image, don't convert to absolute path
+            return input_data
+        return str(utils.abspath(input_data))
```

Comment:
This is solves a different issue which is that remote paths to images were being made absolute when they shouldn't have been.